sr_gui_servoing/src/sr_gui_servoing/displaying.py

```diff
@@ -37,7 +38,7 @@ def __init__(self, node_name):
 
         self.utils = Utils()
 
-        self.image_sub = rospy.Subscriber("/camera/rgb/image_color", Image, self.display)
+        self.image_sub = rospy.Subscriber(self.camera, Image, self.display)
```

Comment:
do this instead (it's much more ROSsy):
 - subscribe to `~/image_color`
 - in the launch file (or from the command line) remap the topic to the proper topic